### PR TITLE
JS error in script-expand-links working example, issue 817

### LINKED ISSUE #817: JavaScript error on working example (script-expand-links)
When trying to see the effects of the [script-expand-links working example](https://www.w3.org/WAI/WCAG21/working-examples/script-expand-links/), I got a JS error in my console that prevented it from working (on Win FF, Chrome, and Edge).

It's choking on line 16 (`for each (link in links)`) since it seems that `for each...in` [has been deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for_each...in). I tried a fix trying to be new school (`for (const link of links)`, I think) but it choked on Edge, then tried `for (const link of Array.from(links))` but that still choked on IE11, so I'll be by shortly with a pull request using old-school `for`.

### PR DESCRIPTION
**Pull request by maxstarkenburg**
Should fix #817 

As noted in the issue, I only had luck in all the browsers I tried using a regular `for` loop, and as noted in the commit message, I didn't actually test the code in the techniques file (just copied and pasted the same diff from the working example).

### CODE REVIEW

File: techniques/client-side-script/SCR30.html [hunk 1]
```diff
@@ -22,10 +22,10 @@
 function doExpand() {
 	var links = document.links;
 	
-	for (link of links) {
```
Comment (patrickhlauke) [inline techniques/client-side-script/SCR30.html]: could probably just change this to a `links.forEach(link) {` without touching the rest of the code

File: working-examples/script-expand-links/index.html [hunk 2]
```diff
@@ -13,10 +13,10 @@
 function doExpand() {
 	var links = document.links;
 	
-	for each (link in links) {
```
Comment (patrickhlauke) [inline working-examples/script-expand-links/index.html]: ditto

File: techniques/client-side-script/SCR30.html (hunk 1 above)
Comment (patrickhlauke) [reply techniques/client-side-script/SCR30.html]: actually, just tried, and seems `links` isn't a proper array but a collection...so yeah, just ignore. better to just be explicit with a for loop...

Comment (maxstarkenburg) [reply techniques/client-side-script/SCR30.html]: Yeah, I had no luck w/ `forEach` either, but I did just push another commit that hopefully makes the overall change a bit more terse.

**Comment by alastc** [comment]
The [preview](https://raw.githack.com/maxstarkenburg/wcag/issue817-expand-links-example-broken/working-examples/script-expand-links/index.html) works for me...

**Comment by alastc** [comment]
Thank you for solving that @maxstarkenburg. As it's solving a bug and not changing the meaning of the technique I can just merge this in. (Other types of changes need to go through more process.)

**Pull request merged**